files/image_config/hostcfgd/hostcfgd

```diff
@@ -25,6 +26,20 @@ TACPLUS_SERVER_PASSKEY_DEFAULT = ""
 TACPLUS_SERVER_TIMEOUT_DEFAULT = "5"
 TACPLUS_SERVER_AUTH_TYPE_DEFAULT = "pap"
 
+global_lock = None
```

Comment:
iso of `global_lock`, can we move this lock to the` AaaCfg()` ? To prevent another classes in this file from using this lock

Reply:
If anyone else (_another subscriber_) gets a need, they could use this.